tests/integration/test_gitlab.py

```diff
@@ -143,7 +140,7 @@ def test_full_repo_name(self):
 
     def test_get_owners(self):
         owners = self.project.get_owners()
-        assert {"lachmanfrantisek", "lbarcziova"} == set(owners)
+        assert set(("lachmanfrantisek", "lbarcziova")).issubset(set(owners))
```

Comment:
```suggestion
        assert {"lachmanfrantisek", "lbarcziova"} <= set(owners)
```

https://docs.python.org/3.7/library/stdtypes.html#frozenset.issubset